README.md

```diff
@@ -85,6 +86,23 @@ Check Docker docs [here](https://github.com/moscajs/aedes-cli#docker)
 
 - [Examples](./docs/Examples.md)
 
+## Clusters
+
+Aedes needs on disk dbs like MongoDB and Redis in order to work with clusters. Based on our tests and users reports the best performances/stability are reached when using [aedes-persistence-mongodb] paired with [mqemitter-redis].
+
+Other info:
+
+- [aedes-persistence-redis] have some memory leak. Check [this issue](https://github.com/moscajs/aedes-persistence-redis/issues/73#issuecomment-686306508)
```

Comment:
I do not think so, as we still do not have a full repro for that issue. A misuse of QoS 1 features can easily look like a memory leak, and we still do not have a repro.